package pack;
import java.io.BufferedReader;
import java.util.*;
import java.io.BufferedReader;
import java.io.*;


public class Exception1 {

	public static void main(String[] args) throws Exception{
/*try{
	int a,b,c;
Scanner s=new Scanner(System.in);
System.out.println("enter a b ");
a=s.nextInt();
b=s.nextInt();
c=a/b;
System.out.println("result:"+c);
}
catch(ArithmeticException e)
{
	System.out.println("errer:try others");
	System.out.println("errer:try others"+e.getMessage());

}*/
		
		/*try {
		int a,b,c;
		BufferedReader br=new BufferedReader(new InputStreamReader(System.in));
		System.out.println("enter a b ");
		a=Integer.parseInt(br.readLine());
		b=Integer.parseInt(br.readLine());
		c=a/b;
		System.out.println("result:"+c);
		}
catch(ArithmeticException e)
{
	System.out.println("errr"+e);
}
catch(Exception e)
{
	System.out.println("error"+e);
}*/

		
	}
}


